application/lib/atom_widget.dart

```diff
@@ -112,9 +111,9 @@ class _AtomWidgetState extends State<AtomWidget> with SingleTickerProviderStateM
                       child: widget.icon,
                     ),
                   widget.label ?? AnimatedBuilder(
-                    animation: widget.atom,
-                    builder: (BuildContext context, Widget child) {
-                      return makeTextForIdentifier(context, widget.atom.identifier);
+                    animation: widget.atom!,
+                    builder: (BuildContext context, Widget? child) {
+                      return makeTextForIdentifier(context, widget.atom!.identifier!);
```

Comment:
line 92 implies atom can be null, but line 114 implies it can't be. we should be consistent.

Reply:
line 51 asserts that label or atom is non-null, and label is null when we execute this code

Reply:
clever